.gitignore

```diff
@@ -1,48 +1,2 @@
-# Compiled Object files
-*.slo
-*.lo
-*.o
-*.obj
-
-# annoying files
-**/.DS_Store
```

Comment:
These are still annoying.

Reply:
@philburk These artifacts are specific to your platform, not to the PortAudio project. Therefore, I would argue, you should try to handle them on your system, not in the PortAudio repo. I recommend setting up a global Git ignore file with `core.excludesfile` in your home directory.

See https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_excludesfile.

Reply:
These can be created on any Mac. If any Mac user does not configure their global config then we get their files. This is a simple and harmless solution.

Reply:
I would say it is "mostly harmless".

> If any Mac user does not configure their global config then we get their files.

I don't think this is a problem in practice.

It is always a possibility that contributors making a PR commit files they didn't intend to. This should be caught in code review.

I think inadventently committing `.DS_Store` files may indeed be a problem when doing the initial commit, but I think it is highly unlikely to ever happen in an already established project.
I have never ever seen a PR where this happened.

And if this *does* actually happen (or you just want to be extra sure that it doesn't ever happen), my advice from above (https://github.com/PortAudio/portaudio/pull/434#discussion_r559217906) should be added to the contributing guidelines. This is real good advice (if I may say so myself) that it worth spreading!